packages/edit-widgets/src/components/sidebar/index.js

```diff
@@ -128,9 +134,28 @@ export default function Sidebar() {
 		<ComplementaryArea
 			className="edit-widgets-sidebar"
 			header={
-				<ComplementaryAreaHeader
-					activeComplementaryArea={ currentArea }
-				/>
+				<ul>
+					<li>
+						<ComplementaryAreaTab
+							identifier={ WIDGET_AREAS_IDENTIFIER }
+							label={
+								selectedWidgetAreaBlock
+									? selectedWidgetAreaBlock.attributes.name
+									: __( 'Widget Areas' )
```

Comment:
Nitpick: Maybe just `selectedWidgetAreaBlock?.attributes?.name || __( 'Widget Areas' )`?

Reply:
I'm not sure that seems right, wouldn't that return the `attributes` object if `name` is not defined?

Reply:
I think I prefer the explicitness that `?:` gives. Also if `attributes.name` is `''` we can see easily that there's a bug.